### DISCUSSION TITLE
sn-scriptsync support for Scripts inside Flow designer

### DISCUSSION THREAD

**Discussion by XikoCat** [arnoudkooi/SN-Utils#400] [Ideas]
I've noticed that sn-scriptsync doesn't work when I'm working with SN's Flow Designer. More specifically, Script Steps inside Actions.
Is there any support planned for it?

**Comment by arnoudkooi** [comment]
If you have save a action, you should get a button on the top of the action, see https://twitter.com/sn_utils/status/1528586520700981249
Does that work?

**Reply by XikoCat** [reply]
I see it! I was blind I guess...
Thank you so much for the fast response and for the fantastic work with this plugin!
It really makes my job a lot easier.

**Comment by XikoCat** [comment]
I see It's still a beta function, how stable is it at this point?
I will be testing it though.
